Modules/CustomRolesHelper.cs

```diff
@@ -148,7 +148,7 @@ public static class CustomRolesHelper
                 CustomRoles.SoulCollector => CustomRoles.Crewmate,
                 CustomRoles.Crewpostor => CustomRoles.Engineer,
                 CustomRoles.Observer => CustomRoles.Crewmate,
-                CustomRoles.DovesOfNeace => CustomRoles.Engineer,
+                CustomRoles.pacifist => CustomRoles.Engineer,
```

Comment:
It would be better if all roles had the first capital letters

Reply:
oops-